### PR TITLE
blsd: init at unstable-2017-07-27

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by orivej** [comment]
Are you still using `blsd`? (I know that you have posted this just 5 days ago.)

**Comment by magnetophon** [comment]
It's a utility you don't need often, so tbh I've mostly tested it as opposed to used.
I do think it's a useful thing to have, assuming that's why you're asking.
Not super important either.

File: pkgs/tools/misc/blsd/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, libgit2 }:
+
+buildGoPackage rec {
+  name = "blsd-${version}";
+  version = "unstable-2017-07-27";
```
Comment (orivej) [inline pkgs/tools/misc/blsd/default.nix]: The version begins with a digit, so unstable is not part of the version, but the suffix of the name. (And for packages which are unlikely to ever have a version number it is fine to drop it.)

File: pkgs/tools/misc/blsd/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  goDeps = ./deps.nix;
+
+  nativeBuildInputs = [ pkgconfig libgit2 ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/junegunn/blsd;
+    description = "List directories in breadth-first order";
+    license = licenses.mit;
+    platforms = platforms.unix;
+  };
```
Comment (orivej) [inline pkgs/tools/misc/blsd/default.nix]: Please list maintainers.

**Review by orivej — COMMENTED** [review]
It could have been handy for `fzf` as the author shows on the home page, but it lists directories not in sorted order which makes it feel unfinished and not something you'd want to use.

Nevertheless it's fine to merge. I've asked in case you've changed your mind, and because maintaining a dependent of libgit is not zero effort.

**Comment by magnetophon** [comment]
Yeah, it's for fzf. I fixed your concerns.
If it really turns out a pain to maintain, I might drop it in the future, so if that's a reason not to merge, I understand.

**Comment by orivej** [comment]
@GrahamcOfBorg eval

**Pull request merged**